### PR TITLE
Add 12.72 monsters

### PR DESCRIPTION
**Pull request by guispiller**
# Description

This PR is to include some missing monsters from Tibia 12.72 version, with their lua script, spells and prey/bestiary inclusion.
I tried to replicate all of the combat features from the creatures, following youtube videos and informations from https://tibia.fandom.com/wiki.

Please delete options that are not relevant.

  - [X] New feature (non-breaking change which adds functionality)

## How Has This Been Tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

  - [X] Summoning all monsters to check spells and appearences
  - [X] Killing all monsters to check body and loot

**Test Configuration**:

  - Server Version: 1.2.0
  - Client: 12.85
  - Operating System: Windows 10

## Checklist

  - [X] My code follows the style guidelines of this project
  - [X] I have performed a self-review of my own code
  - [X] I checked the PR checks reports
  - [ ] I have commented my code, particularly in hard-to-understand areas
  - [ ] I have made corresponding changes to the documentation
  - [X] My changes generate no new warnings
  - [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by majestyotbr** [comment]
Good pull request, thanks for your contribution.
Anyone to help to test?

**Review by majestyotbr — APPROVED** [review]

File: data/monster/humans/carnisylvan_sapling.lua [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+	lookFeet = 0,
+	lookAddons = 0,
+	lookMount = 0
+}
+
+
+monster.health = 750
+monster.maxHealth = 750
+monster.race = "blood"
+monster.corpse = 41720
```
Comment (majestyotbr) [inline data/monster/humans/carnisylvan_sapling.lua]: Check corpse id.

File: data/monster/magicals/bashmu.lua [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+	Stars = 4,
+	Occurrence = 1,
+	Locations = "Salt Caves"
+	}
+
+
+monster.health = 8200
+monster.maxHealth = 8200
+monster.race = "blood"
+monster.corpse = 36804
```
Comment (majestyotbr) [inline data/monster/magicals/bashmu.lua:32]: Need to register corpse id on items.xml

File: data/monster/magicals/girtablilu_warrior.lua [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+	CharmsPoints = 50,
+	Stars = 4,
+	Occurrence = 1,
+	Locations = "Ruins of Nuur"
+	}
+
+monster.health = 8500
+monster.maxHealth = 8500
+monster.race = "blood"
+monster.corpse = 36800
```
Comment (majestyotbr) [inline data/monster/magicals/girtablilu_warrior.lua:32]: Need to register corpse id on items.xml

File: data/monster/magicals/juvenile_bashmu.lua [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+	toKill = 2500,
+	FirstUnlock = 100,
+	SecondUnlock = 1000,
+	CharmsPoints = 50,
+	Stars = 4,
+	Occurrence = 1,
+	Locations = "Salt Caves"
+	}
+
+
```
Comment (majestyotbr) [inline data/monster/magicals/juvenile_bashmu.lua]: Line contains only whitespace

File: data/monster/magicals/juvenile_bashmu.lua [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+	Stars = 4,
+	Occurrence = 1,
+	Locations = "Salt Caves"
+	}
+
+	
+monster.health = 7500
+monster.maxHealth = 7500
+monster.race = "blood"
+monster.corpse = 36967
```
Comment (majestyotbr) [inline data/monster/magicals/juvenile_bashmu.lua:32]: Need to register corpse id on items.xml

File: data/monster/magicals/venerable_girtablilu.lua [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+	CharmsPoints = 50,
+	Stars = 4,
+	Occurrence = 1,
+	Locations = "Ruins of Nuur"
+	}
+
+monster.health = 8500
+monster.maxHealth = 8500
+monster.race = "blood"
+monster.corpse = 36963
```
Comment (majestyotbr) [inline data/monster/magicals/venerable_girtablilu.lua:32]: Need to register corpse id on items.xml

File: data/monster/magicals/venerable_girtablilu.lua [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+	{name = "cyan crystal fragment", chance = 5210, maxCount = 3},
+	{name = "scorpion charm", chance = 5210, maxCount = 1},	
+	{id = 3039, chance = 4910, maxCount = 1}, -- red gem
+	{name = "old girtablilu carapace", chance = 4760, maxCount = 1},
+	{name = "violet gem", chance = 4170, maxCount = 1},
+	{name = "northwind rod", chance = 3570},
+	{name = "wand of cosmic energy", chance = 2680},
+	{name = "blue crystal shard", chance = 2530},
+	{name = "red crystal fragment", chance = 2530},
+	{name = "violet crystal shard", chance = 2530},
```
Comment (majestyotbr) [inline data/monster/magicals/venerable_girtablilu.lua]: Line contains trailing whitespace

File: data/monster/magicals/venerable_girtablilu.lua [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+	{name = "red crystal fragment", chance = 2530},
+	{name = "violet crystal shard", chance = 2530},	
+	{name = "yellow gem", chance = 2530},
+	{name = "underworld rod", chance = 2080},
+	{name = "wand of voodoo", chance = 2080},
+	{name = "blue gem", chance = 1930},
+	{id = 23529, chance = 1930}, -- ring of blue plasma
+	{name = "green crystal fragment", chance = 1640},
+	{name = "green crystal shard", chance = 1640},
+	{name = "wand of defiance", chance = 1340},
```
Comment (majestyotbr) [inline data/monster/magicals/venerable_girtablilu.lua]: Line contains trailing whitespace

File: data/monster/vermins/afflicted_strider.lua [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+	race = BESTY_RACE_VERMIN,
+	toKill = 2500,
+	FirstUnlock = 100,
+	SecondUnlock = 1000,
+	CharmsPoints = 50,
+	Stars = 4,
+	Occurrence = 1,
+	Locations = "Antrum of the Fallen"
+	}
+
```
Comment (majestyotbr) [inline data/monster/vermins/afflicted_strider.lua]: Line contains only whitespace

File: data/monster/vermins/afflicted_strider.lua [hunk 10]
```diff
@@ -0,0 +1,137 @@
...
+	toKill = 2500,
+	FirstUnlock = 100,
+	SecondUnlock = 1000,
+	CharmsPoints = 50,
+	Stars = 4,
+	Occurrence = 1,
+	Locations = "Antrum of the Fallen"
+	}
+	
+
```
Comment (majestyotbr) [inline data/monster/vermins/afflicted_strider.lua:28]: Line contains only whitespace

File: data/monster/vermins/lavafungus.lua [hunk 11]
```diff
@@ -0,0 +1,143 @@
...
+}
+
+monster.attacks = {
+	{name ="melee", interval = 2000, chance = 100, minDamage = 0, maxDamage = -810},
+	{name ="combat", interval = 2000, chance = 50, type = COMBAT_DEATHDAMAGE, minDamage = -560, maxDamage = -650, length = 6, spread = 0, effect = CONST_ME_MORTAREA, target = false},
+	{name ="combat", interval = 2750, chance = 40, type = COMBAT_FIREDAMAGE, minDamage = -490, maxDamage = -720, range = 5, shootEffect = CONST_ANI_FIRE, target = true},
+	{name ="combat", interval = 2750, chance = 30, type = COMBAT_DEATHDAMAGE, minDamage = -720, maxDamage = -810, range = 5, shootEffect = CONST_ANI_SUDDENDEATH, effect = CONST_ME_MORTAREA, target = true},
+	{name ="lavafungus ring", interval = 2000, chance = 20, minDamage = -450, maxDamage = -610},
+	{name ="lavafungus x wave", interval = 2000, chance = 10, minDamage = -640, maxDamage = -730},
+
```
Comment (majestyotbr) [inline data/monster/vermins/lavafungus.lua]: Line contains only whitespace

File: data/scripts/spells/monster/sapling_explode.lua [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+function removeSapling(cid)
+	local creature = Creature(cid)
+	if not isCreature(creature) then return false end
+	creature:remove()
+end
+
+local spell = Spell("instant")
+
+function spell.onCastSpell(creature, var)
+	local from = creature:getId()
```
Comment (majestyotbr) [inline data/scripts/spells/monster/sapling_explode.lua]: Unused variable 'from'

**Review by majestyotbr — CHANGES_REQUESTED** [review]
Check warnings and register corpse ids on items.xml.

**Review by majestyotbr — APPROVED** [review]

**Comment by garaflaks** [comment]
is there the map with the added monsters?

**Pull request merged**